blocks/gu_spdetails/test/block_gu_spdetails_test.php

```diff
@@ -558,7 +564,7 @@ public function test_return_dates(){
         $expectedAssign2->isdueextended = true;
         $expectedAssign2->startdate = $activity->startdate;
         $expectedAssign2->cutoffdate = $activity->cutoffdate;
-        $expectedAssign2->gradingduedate = $expectedAssign2->duedate + (86400 * 14);
+        $expectedAssign2->gradingduedate = $expectedAssign2->duedate;
```

Comment:
Update the corresponding test scenarios based on the requested updates on the code.

Reply:
Fixed at commit [1b50080](https://github.com/uofg-andf/andfmoodle/pull/184/commits/1b50080a3cab7bf33148cba444d1b1d2f7579d88)